Source/Krypton Toolkit/Shared/Krypton.Toolkit.Suite.Extended.Common/Krypton.Toolkit.Suite.Extended.Common.csproj

```diff
@@ -1,8 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Choose>
+    <When Condition="'$(TFMs)' == 'all'">
+      <PropertyGroup>
+        <TargetFrameworks>net35;net40;net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netcoreapp3.1;net5.0-windows;net6.0-windows</TargetFrameworks>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <TargetFrameworks>net48;netcoreapp3.1;net5.0-windows;net6.0-windows</TargetFrameworks>
```

Comment:
Drop NetCore as well

Reply:
And on all the other projects within Extended.

Reply:
Already done! :)